### DISCUSSION TITLE
[Question]: what happens next - what do you see after  you "request a new model" in Frigate+ ?

### DISCUSSION THREAD

**Discussion by ElectronicBattle** [blakeblackshear/frigate#12934] [Ask A Question] [answered]
### What is your question:

https://docs.frigate.video/plus/first_model/#step-2-submit-a-model-request

When I submit a model and I click on the button on the Frigate + web page, what happens next? Can anyone post a screenshot please?

Do I see a page which says "do you want a model of a (tick as appropriate):"

cat
dog
car
number plate
etc.

in which case all boxes ticked would consume (in this example) 4 out of my 12 annual models.

Or do I simply get told "your model is on its way" and I get my first  global and "locally tuned" model, based partly on my image submissions?

(I want to understand how my 12 models per year get used so I can spread them out if needs be, or use them all at once, whatever it turns out to be).

**Comment by NickM-27** [comment]
You are misunderstanding what a model is. A single model detects all objects. 

When your model is trained you will get an email, you go to the dashboard and get the ID, and then put it into your config https://docs.frigate.video/plus/first_model

**Reply by ElectronicBattle** [reply]
The reason for providing a dozen model requests - is it to to allow for changes in camera position, adding a new camera, fine-tuning again due to changes within existing fields-of-view, moving house etc.?

**Reply by NickM-27** [reply, marked as answer]
Most users models aren't perfect after one training. The false positives will likely improve but you may need to annotate more images, start training out new false positives, etc. also yes if you change furniture, stuff changes when the seasons change, etc

**Reply by ElectronicBattle** [reply]
Excellent - I understand now. Thanks both.
